EIPS/eip-2029.md

```diff
@@ -0,0 +1,61 @@
+---
+eip: 2029
+title: State Rent A - State counters contract
+author: Alexey Akhunov (@AlexeyAkhunov)
+discussions-to: https://ethereum-magicians.org/t/eip-2029-state-counters-contract-change-a-from-state-rent-v3-proposal/3279
+status: Draft
+type: Standards Track
+category: Core
+created: 2019-05-15
+---
+
+<!--You can leave these HTML comments in your merged EIP and delete the visible duplicate text guides, they will not appear and may be helpful to refer to if you edit it again. This is the suggested template for new EIPs. Note that an EIP number will be assigned by an editor. When opening a pull request to submit your EIP, please use an abbreviated title in the filename, `eip-draft_title_abbrev.md`. The title should be 44 characters or less.-->
+
+## Simple Summary
+<!--"If you can't explain it simply, you don't understand it well enough." Provide a simplified and layman-accessible explanation of the EIP.-->
+A smart contract is deployed on all Ethereum networks (including testnets), with the code that simply reads the slot in its storage specified by the
```

Comment:
You don't have to specify on which network this EIP gets implemented. That's what the Meta is for. It will require the network fork block numbers. You could specify that the contracts have to be active before or at a specific `FORK_BLOCKNUMBER`.

Reply:
Thank you, I wanted to emphasize that the address of that contract will be the same everywhere, so it does not need to be configured differently in different networks. Modified now